.github/workflows/main.yml

```diff
@@ -11,20 +11,6 @@ on:
       - canary
 
 jobs:
-  check_install_time:
```

Comment:
Shouldn‘t this go into its own PR? 😇

Reply:
It is failing the build, so just getting rid of it :)